### DISCUSSION TITLE
Any future animations?

### DISCUSSION THREAD

**Discussion by j-vitali** [themesberg/flowbite#877] [Q&A] [answered]
Hi guys, will flowbit introduce **component animations**? like **framer-motion**...

**Comment by instrik** [comment, marked as answer]
yes sure

**Comment by instrik** [comment]
sure as hell

**Reply by zoltanszogyenyi** [reply]
Hey @instrik,

Indeed we will, but as we have a long list of features and roadmap to release it will take some time.

Besides, Tailwind is great by allowing you to apply endless amounts of animations yourself as our primary focus is the primitives of the components.

An animations API will be added eventually.

Thanks,
Zoltan

**Comment by j-vitali** [comment]
can i ask when they'll be added?? at least for important ones.. **dropdown**, **modals** etc... 👀
I use also NExtUI and animations are made with framer-motion, very good ones I'd say.. 😁
